src/main/java/pl/futurecollars/invoicing/db/InMemoryDatabase.java

```diff
@@ -3,15 +3,16 @@
 import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
-import java.util.Map;
 import java.util.UUID;
-import org.springframework.stereotype.Component;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Repository;
 import pl.futurecollars.invoicing.model.Invoice;
 
-@Component
+@Repository
```

Comment:
should be `@Service` or `@Component` - @repository is special kind of component reserved for managing DAO

Reply:
Done